examples/hoomd-blue/Butane_ANN.md

```diff
@@ -35,6 +35,8 @@ rm -rf /tmp/cookies.txt
 
 mkdir -p $PYSAGES_ENV .
 unzip -qquo pysages-env.zip -d $PYSAGES_ENV
+apt-cache policy libcudnn8
+apt install --allow-change-held-packages libcudnn8=8.4.1.50-1+cuda11.6
```

Comment:
Can you move these to the next cell in top of `!update-alternatives ...` (making that cell a bash one of course)